python/cudf/cudf/core/udf/strings_typing.py

```diff
@@ -0,0 +1,197 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.
+
+import operator
+
+from numba import types
+from numba.core.typing import signature as nb_signature
+from numba.core.typing.templates import AbstractTemplate, AttributeTemplate
+from numba.cuda.cudadecl import registry as cuda_decl_registry
+
+from strings_udf._typing import StringView, size_type, string_view
+
+from cudf.core.udf import masked_typing
+from cudf.core.udf._ops import comparison_ops
+from cudf.core.udf.masked_typing import MaskedType
+
+masked_typing.MASKED_INIT_MAP[types.pyobject] = string_view
```

Comment:
As long as by `object` we always mean `string` in the context of cuDF, this should always be right. It could only go wrong if we had a dtype in cuDF marked by `object` that _did not_ reference a `StringColumn`.

Reply:
As long as this is strictly only looking at cudf colum ns and not the arbitrary objects identified in a ufunc, I agree that this looks fine.